pom.xml

```diff
@@ -35,11 +35,11 @@
 
         <!-- On this block, library versions must be kept in sync with the ones in the quarkus-bom: -->
         <!-- Start-->
-        <quarkus.version>3.15.1</quarkus.version>
+        <quarkus.version>3.16.1</quarkus.version>
```

Comment:
Can you please upgrade to 3.16.2?... Got out yesterday.

Reply:
Things are going fast!